docs/application/dotnet/guides/nui/nui-components/CameraView.md

```diff
@@ -0,0 +1,62 @@
+---
+keyword: camera, CameraView, NUI, CameraView.DisplayType
+---
+
+# CameraView
+
+The [CameraView](/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.BaseComponents.CameraView.html) class is the NUI view that displays [camera](../../multimedia/camera.md).
+
+
+## Create a CameraView
+
+The following basic example shows how to create a `CameraView` object:
+
+**Figure: Basic CameraView**
+
+![Basic CameraView](./media/camera_images.png)
+
+Step 1. To use the `CameraView` class, add the following namespaces:

```

Comment:
@JoogabYun, could you please remove the term Step and make it as normal ordered list.
```suggestion
1. To use the `CameraView` class, add the following namespaces:

```